site/app/libraries/database/DatabaseQueries.php

```diff
@@ -5793,10 +5793,6 @@ public function changeQueueToken($token, $queue_code) {
         $this->course_db->query("UPDATE queue_settings SET token = ? WHERE code = ?", [$token, $queue_code]);
     }
 
-    public function getLastQueueUpdate() {
```

Comment:
Travis is failing as this function is still used in one other place. Should we be keeping this function then, or refactor the one place using it?

> ```
> ------ --------------------------------------------------------------- 
>  Line   models/OfficeHoursQueueModel.php                               
> ------ --------------------------------------------------------------- 
>  181    Call to an undefined method                                    
>         app\libraries\database\DatabaseQueries::getLastQueueUpdate().  
> ------ --------------------------------------------------------------- 
>```

Reply:
this function is not called any more, i just forgot to remove it